arm-datalake-analytics/job/2016-11-01/swagger/job.json

```diff
@@ -0,0 +1,1070 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DataLakeAnalyticsJobManagementClient",
+    "description": "Creates an Azure Data Lake Analytics job client.",
+    "version": "2016-11-01",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true,
+      "internalConstructors": true
+    }
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{accountName}.{adlaJobDnsSuffix}",
+    "parameters": [
+      {
+        "name": "accountName",
+        "description": "The Azure Data Lake Analytics account to execute job operations on.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      },
+      {
+        "$ref": "#/parameters/adlaJobDnsSuffixInPath"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Jobs/{jobIdentity}/GetStatistics": {
+      "get": {
+        "tags": [
+          "Job"
+        ],
+        "operationId": "Job_GetStatistics",
+        "description": "Gets statistics of the specified job.",
+        "parameters": [
+          {
+            "name": "jobIdentity",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
```

Comment:
It is better to be explicit here that it needs to be a GUID to ensure they don't pass in an invalid string. This ensures type validation during method call.